docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -1,22 +1,110 @@
+
+### Social Presence
+
+When sharing your website on various social media, they will show a small preview and extract metadata from it. Normal SPA applications are very poor at this and often result in blank pages with little to no metadata. Social media crawlers cant really parse JavaScript, so they dont know what to do with SPAs.
+
+With SSR, pages are displayed with nice previews, as both the page content it self and meta tag data is present, allowing for social media crawlers to extract what they need.
+
+### Faster initial interaction
+
+Even though the browser still needs to download and run the whole JavaScript bundle, because the page is already rendered, the user can actually start browsing at that point. This can directly result in a lower drop-off ratio, as users hate waiting.
+
+### Removes white page flicker
+
+This is not as big of a deal, but because the page does not need to be rendered on the client, there will not be a white flicker while the page is being being rendered. This can however be easily mitigated with a loading screen or something similar. 
+
+## Problems with SSR
+
+Even though SSR sounds awesome, there are some things that need to be taken in consideration.
+
+### Time to interaction
+
+Even though they can see the website, it does'nt meant it is working. For very dynamic websites, with allot of JavaScript logic driving the UI, this can lead to weird situations where the page is rendered, looks ready, but the app bundle is still downloading, so no JavaScript logic can be executed yet.
+
+### Time to first bite (TTFB) is slower
+
+Because the server has to actually do the rendering, fetch async data and so on, the time it takes for the response to hit the browser is bigger.
+
+### Server costs
+
+Because there is usually a live Node.js instance always on, waiting to render a request, server costs are higher than static file hosting.
+
+Check the [Hosting page](./hosting.md#server-rendered-websites) for more info.
+
+### Server Load
+
+This is directly related to the above. Server load is higher while pages are being rendered, sometimes blocking other operations until the process is done.
+
+### Window not available
+
+Because the app is rendered in server environment, there is no `window` object. This means libraries like sliders and carousels may not work, and will throw errors during render. 
+
+To overcome this, try not to touch the `window` object, in places, that are called during serve rendering. With sliders and carousels, you could delay their rendering to after the client receives the page, so it gets rendered on the browser.
+
+### Complex to transform deployed SPA to SSR
+
+Because of the above, already deployed websites are harder to transform to SSR, as there might be allot of plugins or UI components, that rely on the `window` object.
```

Comment:
Yes, this is a real-world problem, but only because not enough ppl/component authors are *aware* of SSR.

Workarounds are easy though (`no-ssr`) but remove the SSR benefits.